saleor/csv/models.py

```diff
@@ -1,9 +1,32 @@
+from django.contrib.postgres.fields import JSONField
 from django.db import models
+from django.utils import timezone
 
 from ..account.models import User
 from ..core.models import Job
+from ..core.utils.json_serializer import CustomJsonEncoder
+from . import ExportEvents
 
 
 class ExportFile(Job):
     created_by = models.ForeignKey(User, related_name="jobs", on_delete=models.CASCADE)
     content_file = models.FileField(upload_to="csv_files", null=True)
+
+
+class ExportEvent(models.Model):
+    """Model used to store events that happened during the export file lifecycle."""
+
+    date = models.DateTimeField(default=timezone.now, editable=False)
+    type = models.CharField(
+        max_length=255,
+        choices=[
+            (type_name.upper(), type_name) for type_name, _ in ExportEvents.CHOICES
```

Comment:
Do we have to unwrap it like that? In few other models where we have `choices` we just do `choices=SomeClass.CHOICES`.

Reply:
It depends. In ExportEvents, as the second argument in CHOICES, there is a description of Enum field. So without wrap, the choices will be like:
```choices=[('export_pending', 'Data export was started.'), ('export_success', 'Data export was completed successfully.')]```
instead of
```choices=[("EXPORT_PENDING", "export_pending"), ("EXPORT_SUCCESS", "export_success")]``` .

I based on `OrderEvent`.

Reply:
Yes, I saw the `OrderEvent` uses the same pattern but I'm wondering why it's done like that.  From the two examples you gave above I thought we want the first one. When using `choices` the second item in the choice tuple is supposed to be a human-readable description.

Reply:
:information_desk_person:
I will change it in ExportEvent.